### PR TITLE
MS Bodley 860: new description

### PR DESCRIPTION
**Pull request by adunning**

### CODE REVIEW

File: collections/Bodl/MS_Bodl_860.xml [hunk 1]
```diff
@@ -62,102 +93,1206 @@
...
+                                 <width min="137" quantity="141">137–41</width>
+                              </dimensions> Ruled in pencil; frame and individual lines pricked on all sides of page. Line spacing 4–4.5 mm.</layout>
+                           <layout columns="2" ruledLines="60"><dimensions type="written" unit="mm">
+                                 <height max="192" min="190">190–92</height>
+                                 <width max="144" min="141">141–44</width>
+                              </dimensions><locus>(Gathering 11 (fols. 86 to 93))</locus> is written with tighter line spacing (2.5-3.5 mm). </layout>
+                        </layoutDesc>
                      </objectDesc>
+                     <handDesc>
+                        <handNote script="textualisNorthern">Gothic textualis, first quarter of the twelfth century, with several shifts in hand over the course of the text. Some of these appear to be the same as those responsible for the first part of the manuscript.</handNote>
```
Comment (holfordm) [inline collections/Bodl/MS_Bodl_860.xml:232]: (here and above)- the date seems early for textualis: would protogothic be more appropriate?

File: collections/Bodl/MS_Bodl_860.xml [hunk 2]
```diff
@@ -62,102 +93,1206 @@
...
+                           <layout columns="1" ruledLines="24"><dimensions type="written" unit="mm">
+                                 <height>208</height>
+                                 <width>87</width>
+                              </dimensions> Verses only within the ruled area, written at the same size as the previous section, but one per line, spaced 9 mm apart. Glosses written in the margin, as the Psalm 1 versification in the first part of the manuscript. Additional verses also added in the margin at several points. Ruled in pencil, with pricking on all sides of the page to define both frame and individual lines (often trimmed on the fore-edge). </layout>
+                        </layoutDesc>
                      </objectDesc>
+                     <handDesc>
+                        <handNote script="textualisNorthern"> Written in gothic textualis, first quarter of the twelfth century. Fols. 94r to 126r are written in one hand, after which this shares the work with at least two other scribes.</handNote>
```
Comment (holfordm) [inline collections/Bodl/MS_Bodl_860.xml:1280]: as above: 13th century 1st quarter?

**Comment by holfordm** [comment]
it might be worth stating whether the writing is above/below top line (if you haven't already: I couldn't see it)

File: collections/Bodl/MS_Bodl_860.xml [hunk 3]
```diff
@@ -62,102 +93,1206 @@
...
                   <msContents>
-                     <msItem class="#versus #biblia #commentarii" n="1" xml:id="MS_Bodl_860-part1-item1">
-                        <title key="work_14079" type="desc">Psalm i</title>
-                        <note>(Latin hexameter version, with glosses)</note>
-                        <textLang mainLang="la">Latin</textLang>
+                     <textLang mainLang="la">Latin</textLang>
+                     <msItem class="#versus #biblia #commentarii">
+                        <locus>(fols. 1r-8v)</locus>
+                        <author key="person_61956730">Samuel Presbiter</author>
+                        <title type="desc">Versification of Psalm 1, with marginal glosses</title>
```
Comment (holfordm) [inline collections/Bodl/MS_Bodl_860.xml:104]: does this need an authority record?

File: collections/Bodl/MS_Bodl_860.xml [hunk 4]
```diff
@@ -62,102 +93,1206 @@
...
                   </msIdentifier>
                   <msContents>
-                     <msItem n="1" xml:id="MS_Bodl_860-part2-item1">
-                        <author key="person_61956730">Samuel Priest</author>
-                        <title key="work_4487">Lectura super Psalmos</title>
+                     <textLang mainLang="la">Latin</textLang>
+                     <msItem>
+                        <locus>(fols. 9r–93v)</locus>
+                        <author key="person_61956730">Samuel Presbiter</author>
+                        <title xml:lang="la">Collecta ex auditis super psalmos in scola magistri Willelmi de Monte</title>
```
Comment (holfordm) [inline collections/Bodl/MS_Bodl_860.xml:168]: work_4486 (?)

File: collections/Bodl/MS_Bodl_860.xml (hunk 3 above)
Comment (adunning) [reply collections/Bodl/MS_Bodl_860.xml:104]: There are no other copies, but I've added the identifiers from Stegmüller and Walther if you think records would be useful.

**Comment by adunning** [comment]
Thank you for catching these points! I've now implemented your suggestions in the file.

**Pull request merged**